app/(auth)/sign-in.tsx

```
@@ -86,25 +86,27 @@
+                flexDirection: 'row',
+                alignItems: 'center',
+                marginRight: 12,
+                height: '100%',
+              }}
+            >
               <CountryPicker
                 countryCode={countryCode}
                 onSelect={country => {
-                  setCountryCode(country.cca2);
+                  setCountryCode(country.cca2 as CountryCode);
                   if (country.callingCode && country.callingCode[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)